chicagoland.pandemicresponsecommons.org/etlMapping.yaml

```diff
@@ -15,37 +15,75 @@ mappings:
+          - name: estimated_full_mech_pediatric_ventilators_per_100000
+          - name: icu_beds
           - name: icu_bed_occupancy_rate
+          - name: icu_in_use_beds_non_covid
+          - name: icu_in_use_beds_covid
+          - name: icu_open_beds
+          - name: lastUpdateEt
+          - name: licensed_all_beds
+          - name: licensed_all_beds_per_1000
+          - name: licensed_all_beds_per_1000_gtr_20
+          - name: licensed_all_beds_per_1000_gtr_65
+          - name: negative
           - name: population
           - name: population_gtr_20
           - name: population_gtr_65
+          - name: recovered
+          - name: region_icu_avail
+          - name: region_icu_capacity
+          - name: region_vents_available
+          - name: region_vents_capacity
+          - name: state_icu_capacity
+          - name: staffed_all_beds
+          - name: staffed_icu_beds
           - name: staffed_all_beds_per_1000
           - name: staffed_all_beds_per_1000_gtr_20
           - name: staffed_all_beds_per_1000_gtr_65
           - name: staffed_icu_beds_per_1000
           - name: staffed_icu_beds_per_1000_gtr_20
           - name: staffed_icu_beds_per_1000_gtr_65
-          - name: licensed_all_beds_per_1000
-          - name: licensed_all_beds_per_1000_gtr_20
-          - name: licensed_all_beds_per_1000_gtr_65
-          - name: estimated_full_mech_ventilators
-          - name: estimated_full_mech_ventilators_per_100000
-          - name: estimated_full_mech_pediatric_ventilators
-          - name: estimated_full_mech_pediatric_ventilators_per_100000
-          - name: lastUpdateEt
+          - name: state_icu_covid_patients
+          - name: state_vent_capacity
+          - name: state_vent_covid_patients
+          - name: state_icu_available
+          - name: state_vents_available
+          - name: state_total_beds
+          - name: state_total_beds_available
+          - name: state_total_beds_used
+          - name: state_pct_hospital_beds_available
+          - name: state_adult_icu_capacity
+          - name: state_icu_open_beds
+          - name: state_icu_beds_used
+          - name: state_icu_open_beds_pct
+          - name: state_covid_pui_patients
+          - name: state_covid_pui_patients_pct
+          - name: state_covid_pui_patients_beds_in_use_pct
+          - name: state_Ventilators_open_pct
+          - name: state_ventilators_in_use
+          - name: state_ventilators_in_use_covid
+          - name: state_ventilators_covid_patients_pct
+          - name: state_covid_patients_non_icu
+          - name: state_total_covid_pui_in_icu
+          - name: state_total_covid_pui_in_hospital
+          - name: state_pct_beds_covid_pui
+          - name: testing
+          - name: total_in_use_beds_non_covid
+          - name: total_in_use_beds_covid
+
         sorted_by: updated_datetime, desc
     parent_props:
       - path: projects[project_code:code]
```

Comment:
line 89 to 92 in new branch should be changed to match with QA etl-mapping, similar to line 131-136
We have this format in QA
projects[project_code:code,project_name:name,project_url:url,project_investigator_affiliation:investigator_affiliation]
projects[project_code:code,project_name:name,project_url:url,project_data_description:data_description,project_investigator_affiliation:investigator_affiliation,project_availability_type:availability_type]

Reply:
Yes, I did notice that the format is different in QA, but thought it was a bit cleaner shown this way. The code is exactly the same, just different format.

Reply:
i agree it looks cleaner, but we want the prod config file identical to the QA one

Reply:
I updated according to the feedback.